examples/protocol_specification_ex/output/two_party_negotiation/message.py

```diff
@@ -25,14 +26,34 @@ def __eq__(self, other):
 class TwoPartyNegotiationMessage(Message):
     """A protocol for negotiation over a fixed set of resources involving two parties."""
 
+    protocol_id = "two_party_negotiation"
+
     _speech_acts = {
-        "cfp": {"query": DataModel},
-        "propose": {"query": DataModel, "price": float},
+        "cfp": {
```

Comment:
Ok this is not valid with black. I changed it for a reason and you changed it back. Worth checking in with the other authors of joint work when you see changes and not just change them back.

Reply:
Done